### PR TITLE
Cover basic functionality with tests

### PR DESCRIPTION
**Pull request by freevova**

### CODE REVIEW

**Comment by freevova** [comment]
@EnragedWildkin Please review.

File: test/controllers/user_sessions_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class UserSessionsControllerTest < ActionController::TestCase
+  def test_should_get_new
```
Comment (EnragedWildkin) [inline test/controllers/user_sessions_controller_test.rb]: `test_get_new_renders_new_template` is better in my opinion.

File: test/controllers/users_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,107 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionController::TestCase
+  def test_should_get_index
```
Comment (EnragedWildkin) [inline test/controllers/users_controller_test.rb]: `test_get_index_assigns_users`

File: test/controllers/users_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+  def test_should_get_new
+    login(:admin)
+    get :new
+
+    assert_response :success
+    assert_template :new
+  end
+
+  def test_should_get_edit
```
Comment (EnragedWildkin) [inline test/controllers/users_controller_test.rb]: Same as above.

File: test/helpers/contracts_helper_test.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
+require 'test_helper'
+
+class ContractsHelperTest < ActionView::TestCase
+  def test_locale_date_method_returns_date_in_different_languages
+    assert_equal '17 June 2016', locale_date(contract.start_date, "%d %B %Y", :en)
+    assert_equal '17 червня 2016', locale_date(contract.start_date, "%d %B %Y", :ua)
+    assert_equal '17 июня 2016', locale_date(contract.start_date, "%d %B %Y", :ru)
+  end
+
+  def test_today_date_method_returns_today
```
Comment (EnragedWildkin) [inline test/helpers/contracts_helper_test.rb]: `test_today_date` is enough in my opinion.

File: test/helpers/contracts_helper_test.rb [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    assert_equal Time.now.strftime('%Y-%m-%d'), today_date
+  end
+
+  def test_zip_code_method_returns_zip_code
```
Comment (EnragedWildkin) [inline test/helpers/contracts_helper_test.rb]: Same as above.

File: test/helpers/contracts_helper_test.rb [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    assert_equal '88888', zip_code(address)
+  end
+
+  def test_street_method_returns_street_name
```
Comment (EnragedWildkin) [inline test/helpers/contracts_helper_test.rb]: Same as above.

File: test/helpers/contracts_helper_test.rb [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    assert_equal 'Some Str.', street(address)
+  end
+
+  def test_city_method_returns_city_name
```
Comment (EnragedWildkin) [inline test/helpers/contracts_helper_test.rb]: Same as above.

File: test/helpers/contracts_helper_test.rb [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    assert_equal 'Lviv', city(address)
+  end
+
+  def test_country_method_returns_country_name
```
Comment (EnragedWildkin) [inline test/helpers/contracts_helper_test.rb]: Same as above.

File: test/models/user_field_test.rb [hunk 9]
```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class UserFieldTest < ActiveSupport::TestCase
+  def setup
+    @user_field ||= user_fields(:en_john_field)
+  end
+
+  def test_should_return_short_name
```
Comment (EnragedWildkin) [inline test/models/user_field_test.rb]: Same as above.

File: test/models/user_field_test.rb [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+    assert_equal 'Ivanov Ivan', @user_field.short_name
+  end
+
+  def test_should_return_full_name
```
Comment (EnragedWildkin) [inline test/models/user_field_test.rb]: Same as above.

File: test/models/user_field_test.rb [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+    assert_equal 'Ivanov Ivan Ivanovych', @user_field.full_name
+  end
+
+  def test_should_return_name_with_initials
```
Comment (EnragedWildkin) [inline test/models/user_field_test.rb]: Same as above.

File: test/models/user_test.rb [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+    @user = User.create(email: '')
+
+    assert_not @user.valid?
+  end
+
+  def admin
+    @user ||= users(:admin)
+  end
+
+  def user
```
Comment (EnragedWildkin) [inline test/models/user_test.rb]: Is it used anywhere?

File: test/models/user_test.rb [hunk 13]
```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+class UserFieldTest < ActiveSupport::TestCase
+  def test_admin
+    assert admin.admin?
+  end
+
+  def test_skip_login_fields_validation_when_email_blank_and_permission
+    @user = User.new(email: '')
```
Comment (EnragedWildkin) [inline test/models/user_test.rb]: Don't need to use instance variables.

**Comment by EnragedWildkin** [comment]
@freevova Please check my comments.

**Comment by freevova** [comment]
@EnragedWildkin Please review.

**Pull request merged**